templates/operations.html

```diff
@@ -473,6 +475,12 @@ <h1 id="time-tactic" class="member-title"></h1>
                     refreshUpdatableFields(OPERATION.chain[i], existing);
                 }
             }
+            if(OPERATION.state == "running" && OPERATION.chain.length == 0) {
+                stream("No abilities were ran. Click '+ potential links' to add new abilities to run in your operation or click the stop button to finish the operation.");
+            }
+            if(OPERATION.state == "running" && status.every(function(s, idx) { return s != -3 })) {
```

Comment:
If possible, let's try to avoid needing the new `status` variable. Any reason why this line can't use `Operation.chain` directly? 

```
 if(OPERATION.state == "running" && OPERATION.chain.every(function(link, idx) { return link.status != -3 })) {
```

Reply:
No reason, change should be reflected in the most recent commit